pipeline/config/exoscale/sc-config.yaml

```diff
@@ -74,6 +75,7 @@ user:
     - severity
 harbor:
   enabled: true
+  subdomain: harbor
```

Comment:
I think you forgot to add the new subdomains (notary, elastic etc.) to these files (wc-config as well)